### DISCUSSION TITLE
[Livewire V3] Alpine.navigate() is not a function

### DISCUSSION THREAD

**Discussion by cwkhawand** [alpinejs/alpine#3828] [7. Bugs (archived)]
When I try using Livewire's $this->redirect('/page', navigate: true), the browser logs a strange error:

```
Uncaught (in promise) TypeError: Alpine.navigate is not a function
    at shouldRedirectUsingNavigateOr
```

For reference, I only have 1 script importing Alpine on the page (the one Livewire automatically loads), on version 3.13.2.
Any advice is appreciated.

**Comment by ekwoka** [comment]
This isn't part of the Alpine repo.

It's Livewire only.
